queries.sql

```diff
@@ -10,3 +11,77 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
 
+/* Tuesday's queries and transactions */
+
+-- TRANSACTION #1
+BEGIN;
+
+-- set set species column to unspecified
+UPDATE animals
+SET species = 'unspecified';
+
+-- roll back the transaction
+ROLLBACK;
+
+
+-- TRANSACTION #2 
+
+BEGIN;
+
+-- set the species column to digimon for all animals that have a name ending in mon
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- set the species column to pokemon for all animals that don't have species already set.
+UPDATE animals
+SET species = 'pokemon' WHERE species IS NULL;
+
+-- commit the transactions
+COMMIT;
+
+-- TRANSACTION #3
+BEGIN;
+
+-- delete all records in the animals table
+DELETE FROM animals;
+
+-- roll back the transaction
+ROLLBACK;
+
+-- TRANSACTION #4
```

Comment:
- [ ] Kindly make sure to verify that change was made before ROLLBACK, and also verify that change was undone after ROLLBACK by adding this (before and after the rollback:
`SELECT COUNT(*) FROM ANIMALS;`